app/Http/Controllers/CampaignController.php

```diff
@@ -5,19 +5,23 @@
 use App\Http\Requests\CampaignStoreRequest;
 use App\Http\Requests\CampaignUpdateRequest;
 use App\Models\Campaign;
+use App\Models\User;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Inertia\Inertia;
 
 class CampaignController extends Controller
 {
     /**
      * @param \Illuminate\Http\Request $request
-     * @return \Illuminate\Http\Response
+     * @return \Inertia\Response
      */
     public function index(Request $request)
     {
-        $campaigns = Campaign::all();
+        $user = User::where('id', Auth::user()->id)->get();
+        $userCampaigns = $user->first()->campaigns()->get();
 
-        return view('campaign.index', compact('campaigns'));
+        return Inertia::render('Campaign/Campaign', compact('userCampaigns'));
```

Comment:
if the `Campaign/Campaign` page component doesn't exist, then you cant do this. You'd also need to change the test to expect an inertia response rather than a view.

Also if you take my suggestion from above, then you'd need to do

```suggestion
        return Inertia::render('Campaign/Campaign', compact('campaigns'));
```

Reply:
Alright, I'll change it back to return a view, since we don't have the campaign page component.